llama_stack/providers/remote/inference/tgi/tgi.py

```diff
@@ -128,6 +128,10 @@ def _build_options(
         fmt: ResponseFormat = None,
     ):
         options = get_sampling_options(sampling_params)
+        # TGI does not support temperature=0, so we set it to 1e-3 instead
+        if options["temperature"] == 0:
+            options["temperature"] = 1e-3
```

Comment:
1e-5 to  be even better? but also `==` is too weak a check for floats? maybe you should do something like `if float(...) < eps: value = eps`

Reply:
1e-5 just gives outputs "!!!!!!!!!!" :(

Reply:
Added a comment for the greedy w/ temperature = 0 case.